src/app/core/signing.guard.ts

```diff
@@ -69,13 +72,16 @@ export class SigningGuard implements CanActivate {
         return true;
       } else {
         // noinspection JSIgnoredPromiseFromCall
-        this.router.navigate([ '/signing-auth-redirect' ], { queryParams: { uid: SigningGuard.uid, key: SigningGuard.key } });
+        this.router.navigate([ '/signing-auth-redirect' ], { queryParams: { uid: SigningGuard.uid, key: SigningGuard.key }} )
+          .catch(e => alert('Something went wrong, see the error below: \n' + e + '\n please contact us at virtrolio.team@gmail.com if this ' +
+            'problem persists.'));
         return false;
       }
     })
       .catch(() => {
         // noinspection JSIgnoredPromiseFromCall
-        this.router.navigate([ '/invalid-link' ]);
+        this.router.navigate([ '/invalid-link' ]).catch(e => alert('Something went wrong, see the error below: \n' + e +
+          '\n please contact us at virtrolio.team@gmail.com if this problem persists.'));
```

Comment:
Replace with standardized Error message
```suggestion
        this.router.navigate([ '/invalid-link' ]).catch(e => AuthService.displayError(e));
```